### ISSUE TITLE
Create view component error 

### ISSUE THREAD

**Issue by ymg2007** [Sonderwoods/MantaRay#98]
I downloaded the MantaRay plugin and tried to run on Rhino and Grasshopper. The previous steps is fine. But the create view compotent ran with error. It gave me the flowing error "solution exception: unkown units". Do you extract the camara postion and target position?
![Rhino_2022-09-08_19-10-31](https://user-images.githubusercontent.com/24608776/189107803-2c7fcac3-74dd-48de-95b3-9c08ccd4a238.png)
I tried to created named viewes in Rhino, but it does not work . Please help me to solve this the solution. 

Thank you @Sonderwoods

**Comment by Sonderwoods** [comment]
Hi mingyang, thanks for testing it out. You might be our first user!! ;-).

Be sure you pulled the latest version and build it yourself in Visual
Studio. If you don't know how to do that then I'll release a new gha file
shortly.

Can you perhaps send me the files you were working with so I can try and
reproduce the error?

ATM the camera component only works in perspective. Will need to look I to
parallel projection (like top view) later.

Best, M

tor. 8. sep. 2022 13.19 skrev minggangyin ***@***.***>:

**Comment by ymg2007** [comment]
Hi sonderwoods. Thank for you reply.
I just used the [v1.0.0.1-alpha] and tested the tutorial geometry file and test connection grasshopper file. 
![image](https://user-images.githubusercontent.com/24608776/189114522-07a496c3-3012-4e19-be3e-969396fa5a12.png)

![image](https://user-images.githubusercontent.com/24608776/189114153-96587633-3a7c-4264-a825-839f9062d43b.png)

**Comment by ymg2007** [comment]
Hi Sonderwoods.

I downloaded newest V1.0.0.2 version and tested it on Rhino and Grasshopper. The problem still here. A little change is the compotent can generate vf file but vf file is empty. 
![image](https://user-images.githubusercontent.com/24608776/189261136-9dabd1c3-5544-4966-a60f-948d97573a30.png)

For command "genbox", I tested it and ran file.
![image](https://user-images.githubusercontent.com/24608776/189260091-eee40952-3392-4675-8c59-bda5f8e24d63.png)

**Comment by ymg2007** [comment]
Hi Sonderwoods. An update for this problem.
It is a big miscake. I found the reason for this. Because the language of my rhino is set as simplified Chinese. So the create camera can not get the units of Millimeter(“毫米”). I changed the language to English, camera component worked file. 
Here is the output image of TestRender Grasshopper file. 
MantaRay is so powerful. And the render is so quick than Windows Radiance version.

![image](https://user-images.githubusercontent.com/24608776/189264807-5af9bc0f-bdb1-44c2-9d5b-4a866ef7af99.png)
